share/spice/sound_cloud/sound_cloud.js

```diff
@@ -65,7 +65,7 @@
                     return;
                 }
 
-                var streamURL = '/sc_audio/' + o.stream_url;
+                var streamURL = '/sc_audio/?u=' + o.stream_url;
```

Comment:
I had to add ?u= to the querystring here to make it work. I don't see how the internal redirect works without that.